nautobot/extras/views.py

```diff
@@ -1554,6 +1555,76 @@ def post(self, request, class_path=None, pk=None):
 class JobView(generic.ObjectView):
     queryset = JobModel.objects.all()
     template_name = "extras/job_detail.html"
+    object_detail_content = object_detail.ObjectDetailContent(
+        panels=[
+            object_detail.ObjectFieldsPanel(
+                weight=100,
+                section=SectionChoices.LEFT_HALF,
+                label="Source Code",
+                fields=[
+                    "module_name",
+                    "job_class_name",
+                    "class_path",
+                    "installed",
+                    "is_job_hook_receiver",
+                    "is_job_button_receiver",
+                ],
+            ),
+            object_detail.ObjectFieldsPanel(
+                weight=100,
```

Comment:
Please make sure weights are unique within a section to ensure consistent ordering.